eclipse-che/che-server
# Add provider name annotation to PAT secret

### What does this PR do?
Add `che.eclipse.org/scm-provider-name` annotation to Personal Access Token secret.

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/22802

### How to test this PR?

1. Deploy che with the PR image: `quay.io/eclipse/che-server:pr-670`
2. Apply the related dashboard image: `quay.io/eclipse/che-dashboard:pr-1075`
3. Setup [GitHub oauth](https://eclipse.dev/che/docs/stable/administration-guide/configuring-oauth-2-for-github/)
4. Start a workspace from a GitHub repository with a devfile, consent the authorisation request. 
5. Go to dashboard -> User Preferences -> Personal Access Tokens tab.

See: The provider value is `github`:
![screenshot-eclipse-che apps 7a178e4601d963f19462 hypershift aws-2 ci openshift [org-2024](https://issues.redhat.com//browse/org-2024) 03 18-15_59_44](https://github.com/eclipse-che/che-server/assets/7668752/8f861750-9795-4b13-9212-ad4847b303c7)


### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
